src/gas-hub.js

```diff
@@ -1,13 +1,36 @@
 'use strict';
 
+let inited = false;
+let currentUrl;
 let gas;
 let scm;
 let context = {};
 const LEVEL_ERROR = 'Warning';
 const LEVEL_WARN = 'Info';
 const LEVEL_INFO = 'Notice';
+const observer = new MutationObserver((e) => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/083dcd77ddb6869d2b7dc2cdf2f7452c410d4901/src/gas-hub.js#L11"></span>‘e’ is defined but never used.</td>
    </tr>
  </tbody>
</table>